package.json

```diff
@@ -169,6 +169,7 @@
   },
   "homepage": "https://github.com/indico/indico#readme",
   "jest": {
+    "transformIgnorePatterns": ["/node_modules/(?!geodesy)"],
```

Comment:
Maybe I missed some discussion, but why are we not ignoring the `geodesy` package?

Reply:
correct me if i missed anything @plourenco, but iirc this package uses *only* ES modules so it's the only one that needs to be transformed atm when running jest even though it's something from `node_modules`

Reply:
Yes, because in `indico/modules/rb/client/js/common/map/util.js` line 14:

```
import LatLon from 'geodesy/latlon-nvector-spherical';
```

We are importing that module directly which was published untranspiled, it is a pure ES6 class. Since all files inside node_modules are not transformed by default, Jest will not understand the code in these modules, resulting in syntax errors.

The same happens with `semantic-ui-react` in #4461, where I'm ignoring it.

Reply:
OK, just wanted to clarify that, thanks!